p2p/peer.go

```diff
@@ -196,7 +200,9 @@ func (p *peer) HandshakeTimeout(ourNodeInfo *NodeInfo, timeout time.Duration) er
 	}
 
 	// Remove deadline
-	p.conn.SetDeadline(time.Time{})
+	if err := p.conn.SetDeadline(time.Time{}); err != nil {
+		return errors.Wrap(err, "Error Removing deadline")
```

Comment:
why above Deadline capitalized and here it is not? better to stick to one style: `Failed to remove deadline`

Reply:
Thanks @melekes for the catch! For some reason I thought I was copying the style around but now looking back at it, the accompanying style doesn't have such capitalizations. I'll update it in a second.

Reply:
@melekes I've force pushed because I wanted to keep the example in the commit message for coherence.